### PR TITLE
fix: make property name consistent #4461

### PR DESCRIPTION
**Pull request by chenghuichen**
### What is the purpose of the change

make property name consistent. 

### Related issues/PRs

Related issues: #4461

### Brief change log

- make property name consistent

### Checklist

- [x] I have read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests).
- [x] I have explained the need for this PR and the problem it solves
- [x] I have explained the changes or the new features added to this PR
- [x] I have added tests corresponding to this change
- [x] I have updated the documentation to reflect this change
- [x] I have verified that this change is backward compatible (If not, please discuss on the [Linkis mailing list](https://linkis.apache.org/community/how-to-subscribe) first)
- [x] **If this is a code change**: I have written unit tests to fully verify the new behavior.

### CODE REVIEW

File: linkis-dist/package/conf/linkis-cg-linkismanager.properties [hunk 1]
```diff
@@ -20,8 +20,8 @@ wds.linkis.server.mybatis.mapperLocations=classpath*:mapper/common/*.xml,classpa
 wds.linkis.server.mybatis.typeAliasesPackage=
 wds.linkis.server.mybatis.BasePackage=org.apache.linkis.manager.dao,org.apache.linkis.manager.rm.external.dao,org.apache.linkis.engineplugin.server.dao
 
-wds.linkis.engineConn.plugin.cache.expire-in-seconds=100000
-wds.linkis.engineConn.dist.load.enable=true
+wds.linkis.engineconn.plugin.cache.expire-in-seconds=100000
```
Comment (peacewong) [inline linkis-dist/package/conf/linkis-cg-linkismanager.properties:23]: need to update EngineConnPluginCacheConfig class

**Comment by chenghuichen** [comment]
The real property name is 
image

and

image

**Review by jacktao007 — APPROVED** [review]
LGTM.

**Review by casionone — APPROVED** [review]

**Review by peacewong — APPROVED** [review]
LGTM.

**Pull request merged**